packages/amazonq/src/app/inline/sessionManager.ts

```diff
@@ -86,4 +87,17 @@ export class SessionManager {
     public clear() {
         this.activeSession = undefined
     }
+
+    // re-render the session ghost text to display paginated responses once per completed session
+    public async maybeRefreshSessionUx() {
+        if (
+            this.activeSession &&
+            !this.activeSession.isRequestInProgress &&
+            !this._refreshedSessions.has(this.activeSession.sessionId)
+        ) {
+            await vscode.commands.executeCommand('editor.action.inlineSuggest.hide')
+            await vscode.commands.executeCommand('editor.action.inlineSuggest.trigger')
+            this._refreshedSessions.add(this.activeSession.sessionId)
+        }
```

Comment:
@leigaol so this will only be invoked when cx click ">" or "<" correct? So the suggestions in the pagination response will only be rendered once they press either keyboard shortcut right?

Reply:
Yes. Do not render paginated response if there is no Left Right key input.